WordPress/src/main/java/org/wordpress/android/ui/pages/PagesActivity.kt

```diff
@@ -1,12 +1,19 @@
 package org.wordpress.android.ui.pages
 
+import android.app.Activity
+import android.content.Intent
 import android.os.Bundle
 import android.support.v7.app.AppCompatActivity
 import android.view.MenuItem
-import kotlinx.android.synthetic.main.toolbar.*
+import kotlinx.android.synthetic.main.pages_fragment.*
 import org.wordpress.android.R
+import org.wordpress.android.R.id
+import org.wordpress.android.ui.RequestCodes
+import org.wordpress.android.ui.posts.BasicFragmentDialog.BasicDialogNegativeClickInterface
+import org.wordpress.android.ui.posts.BasicFragmentDialog.BasicDialogPositiveClickInterface
+import org.wordpress.android.ui.posts.EditPostActivity
 
-class PagesActivity : AppCompatActivity() {
+class PagesActivity : AppCompatActivity(), BasicDialogPositiveClickInterface, BasicDialogNegativeClickInterface {
```

Comment:
I really don't like `BasicFragmentDialog` and the way it passes clicks through the activity. This is coupling Activity and Fragment. I'd just pass in callback (to the `BasicFragmentDialog`. Not much you can do here, but this is (for example) making bottom nav harder.